grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package dataplane;
+
+option go_package = "dataplane/grpc"; // Adjust based on your project
+
+service DataplaneService {
+    rpc GetGlobalConfig(GetGlobalConfigRequest) returns (GlobalConfigResponse);
+
+    rpc CheckResourceVersion(CheckResourceVersionRequest) returns (CheckResourceVersionResponse);
+
+    rpc UpdateGlobalConfig(UpdateGlobalConfigRequest) returns (GlobalConfigResponse);
+}
+
+message GlobalConfig {
+    string generation = 1;
+    repeated Vpc vpcs = 2;
+    repeated PeeringInterface pifs = 3;
+    repeated PeeringPolicy policies = 4;
+}
+
+message Vpc {
+    string name = 1;
+    repeated Subnet subnets = 2;
+}
+
+message Subnet {
+    string name = 1;
+    string cidr = 2;
+    uint32 vni = 3;
+}
+
+message PeeringInterface {
```

Comment:
Let's not merge it until we agree on the GW API as it can affect PIFs/Policies significantly